stdlib/Printf/src/Printf.jl

```diff
@@ -409,11 +411,13 @@ const __BIG_FLOAT_MAX__ = 8192
                 if len > siz
                     maxout = max(__BIG_FLOAT_MAX__,
                                  ceil(Int, precision(x) * log(2) / log(10)) + 25)
+                    # TODO: `error` is kind of generic, is there a more appropriate one to throw?
                     len > maxout &&
                         error("Over $maxout bytes $len needed to output BigFloat $x")
                     resize!(buf, len + 1)
                     len = _snprintf(pointer(buf, pos), len + 1, str, x)
                 end
+                # TODO: when will this actually be thrown? What more informative error would be appropriate?
                 len > 0 || throw(ArgumentError("invalid printf formatting $str for BigFloat"))
```

Comment:
I'm assuming this is then only hit when our `BigFloat` is larger than `__BIG_FLOAT_MAX__` ?